public/framework/core_refs.ts

```diff
@@ -34,6 +36,8 @@ class CoreRefs {
   public dashboardProviders?: unknown;
   public overlays?: OverlayStart;
   public dataSource?: DataSourcePluginStart;
+  public dataSources?: DataSourceStart;
```

Comment:
Both of these coreRefs have very similar names, is there plans to change the naming?